cfme/services/catalogs/catalog.py

```diff
@@ -99,8 +99,8 @@ def create(self):
         })
         view.add_button.click()
         view = self.create_view(CatalogsView)
-        view.flash.assert_success_message('Catalog "{}" was saved'.format(self.name))
         assert view.is_displayed
+        view.flash.assert_success_message('Catalog "{}" was saved'.format(self.name))
```

Comment:
Required: Flash messages text is checked in tests, assert_no_error() is good for methods